sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -262,4 +262,12 @@ object StaticSQLConf {
       .stringConf
       .toSequence
       .createWithDefault(Nil)
+
+  val DISABLED_JDBC_CONN_PROVIDER_LIST =
+    buildStaticConf("spark.sql.sources.disabledJdbcConnProviderList")
+      .doc("Configures a list of JDBC connection providers, which are disabled. " +
+        "The list contains the name of the JDBC connection providers separated by comma.")
+      .version("3.1.0")
```

Comment:
Is it better to describe this update in the migration guide?

Reply:
I think it's okay ... I guess ... it was already no-op if somebody sets in runtime and it was documented as a static configuration already too .. I would think this is a bug ..